bin/cfg/Tomb1Main.json5

```diff
@@ -61,15 +61,17 @@
     "enable_smooth_bars": true,
 
     // Changes how the healthbar is displayed. Possible values:
-    // - always: always show the healthbar
-    // - default: show the healthbar at the beginning of a level, after
-    //   getting hit or while having weapons equipped (as in OG)
-    // - flashing-or-default: show the healthbar when Lara's health is 20% or
-    //   below, or under default circumstances
+    // - default: show the healthbar at the start of a level, after getting
+    //   hit, or while having weapons equipped; flash the healthbar if it's already
+    //   on screen when Lara's health is 20% or below (TombATI)
+    // - flashing-or-default: constantly flash the healthbar when Lara's
+    //   health is 20% or below, or show under default circumstances
     // - flashing-only: show the healthbar *only* when Lara's health is 20% or
-    //   below (useful for challenge runs)
+    //   below (for challenge runs)
+    // - always: always show the healthbar
     // - never: never display the healthbar (for challenge runs)
-    "healthbar_showing_mode": "flashing-or-default",
+    // - ps1: show the healthbar under default circumstances with no flashing
+    "healthbar_showing_mode": "default",
```

Comment:
I think we agreed to keep it as `flashing-or-default`; from yesterday's comment:
>By default I would like to have the healthbar shown when Lara's health changes, or if it's under 20% in which case I'd like it to flash. This is `flashing-or-default`

Reply:
Ya sorry. Changed it to TombATI and forgot to change it back.